RELEASE_NOTES.md

```diff
@@ -0,0 +1,4 @@
+# DateBuilder Release notes
+
+## February 11th 2025
+- Initial version published
```

Comment:
Please fixup this commit with the main one of the PR

Reply:
ok